.github/workflows/ci-hobbies-helsinki-check.yml

```diff
@@ -76,7 +76,6 @@ jobs:
         run: |
           yarn install --immutable --inline-builds
         env:
-          PRISMA_SKIP_POSTINSTALL_GENERATE: true
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
```

Comment:
playwright isnt used in hobbies, so all references to it could be removed from monorepo?

Reply:
Playwright relations removed